.github/workflows/release-federation.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: release-federation
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+env:
+  REGISTRY_NAME: registry.scality.com
+  PROJECT_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY_NAME }}
+          username: ${{ secrets.REGISTRY_LOGIN }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: images/svc-base/Dockerfile
+          push: true
+          tags: "${{ env.REGISTRY_NAME }}/${{env.PROJECT_NAME }}/${{ env.PROJECT_NAME }}:${{ github.event.inputs.tag }}-svc-base"
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
You could save most of this by using shared worfklow :
https://github.com/scality/workflows/blob/main/.github/workflows/docker-build.yaml

Then it would look something like : 

```
  steps: 
    uses: scality/workflows/.github/workflows/docker-build.yaml@v1
    secrets: inherit
    with:
      push: true
      namespace: ${{ env.PROJECT_NAME }}
      name: ${{ env.PROJECT_NAME }}
      tag: "${{ github.event.inputs.tag }}-svc-base"
      context: images/svc-base/Dockerfile
      registry: ${{ env.REGISTRY_NAME }}
```

Reply:
This was not possible to use as the `file` input was not being used in that workflow, it is now so switching over.